src/deavinaSJun20/CheckStringIsPalindrome.java

```diff
@@ -0,0 +1,26 @@
+package deavinaSJun20;
+
+import java.util.Scanner;
+
+public class CheckStringIsPalindrome {
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Please Enter the String : ");
+		String input = scanner.next();
+		new CheckStringIsPalindrome().checkStringIsPalindrome(input);
+		scanner.close();
+	}
+
+	void checkStringIsPalindrome(String name) {
+		String reverse = "";
+		String name1 = name;
+		for (int i = name.length() - 1; i >= 0; i--) {
+			reverse = reverse + String.valueOf(name.charAt(i));
+		}
+		if (name1.equals(reverse))
```

Comment:
Here while comparing Strings, practice equalsIgnoreCase instead of equals

Reply:
made changes as per review comments and updated file checked in

Reply:
No Jagdeesh here we should use equals only.